### PR TITLE
Bug/marketplace page dynamic page #27

### LINKED ISSUE #27: marketplace page still using a sample data, make page as a dynamic page
## Problem 
marketplace page still using a sample data, make page as a dynamic page

## Evidence
image

## Bug fix
- Display NFT's image using NTF's image URL. ※Green area
- Display NFT token information ※Red area
- Save contract info and market url to DynamoDB

### PR DESCRIPTION
**Pull request by minhtuan412**
Create api save contract info to dynamicDB

fix #27

### CODE REVIEW

File: api/src/post.js [hunk 1]
```diff
@@ -86,4 +86,52 @@ router.get("/:tweet_id", async (req, res) => {
...
+
+    await dynamodb
+      .put({
+        TableName: "Contract",
+        Item: tweet,
+      })
+      .promise();
+
+    console.log(res);
+    res.json({ tweet_id: tweet.tweet_id, url: tweet.url, name: tweet.name, transaction_hash: tweet.transaction_hash });
```
Comment (hanchenglin) [inline api/src/post.js:108]: @minhtuan412 Please also save contract address, token id

File: api/src/post.js [hunk 2]
```diff
@@ -86,4 +86,52 @@ router.get("/:tweet_id", async (req, res) => {
...
+      date: systemDate,
+      url: req.url,
+      name: req.name,
+      transaction_hash: req.transactionHash
+    };
+
+    await dynamodb
+      .put({
+        TableName: "Contract",
+        Item: tweet,
```
Comment (hanchenglin) [inline api/src/post.js:103]: @minhtuan412  Please Item column name is not good! 
You can save 
- contract_address
- token_id
- name
- url
- transaction_hash
- date
- tweet_id
- updated_at
- created_at

**Comment by hanchenglin** [comment]
@minhtuan412  Please merge develop branch into your work branch!

File: api/src/post.js [hunk 3]
```diff
@@ -86,4 +86,78 @@ router.get("/:tweet_id", async (req, res) => {
...
+    const contract = dbUser.Item;
+
+    res.json(contract);
+  } catch (e) {
+    console.log(e);
+    res.status(500).send(e);
+  }
+});
+
+router.get("/contract/getAll", async (req, res) => {
```
Comment (hanchenglin) [inline api/src/post.js:141]: @minhtuan412 /contract/getAll →/contract

**Comment by hanchenglin** [comment]
@minhtuan412  Please fix conflict!

File: api/src/post.js (hunk 1 above)
Comment (minhtuan412) [reply api/src/post.js]: I updated it

File: api/src/post.js (hunk 2 above)
Comment (minhtuan412) [reply api/src/post.js]: I update it

File: api/src/post.js (hunk 3 above)
Comment (minhtuan412) [reply api/src/post.js:141]: @hanchenglin I updated but I check again, I see call api get return status 200, data return empty

Comment (minhtuan412) [reply api/src/post.js:141]: image
image

**Comment by hanchenglin** [comment]
@minhtuan412 Please fix conflict!

**Comment by minhtuan412** [comment]
> @minhtuan412 Please fix conflict!

@hanchenglin I fixed it

**Pull request merged**